### DISCUSSION TITLE
Aurawatch Cooldown sorting

### DISCUSSION THREAD

**Discussion by Xerolution** [siweia/NDui#163] [Q&A] [answered]
Hi,

is it possible to sort the icons displayed by aurawatch?
i added some speels to the cooldown aurawatch and would like to sort them by remaining cooldown.
it would me much easier to know which spell is aviable next if they are in the right order.

quick example:
right now its like that:
Immolation Aura       20 sec
Death Sweep             5 sec
Eye Beam                  25 sec

i would prefer it like that:
Eye Beam                  25 sec
Immolation Aura       20 sec
Death Sweep             5 sec

PS: thanks for the grat work and the excellent ui!
Cheers!

**Comment by siweia** [comment, marked as answer]
Sorry, not possible to do the way you said.
